lib/compiler/passes/generate-js.js

```diff
@@ -1414,7 +1485,17 @@ function generateJS(ast, options) {
           parts.push("");
         }
 
+        for (let i = 0; i < ast.imports.length; i++) {
```

Comment:
nit: this is slightly inefficient. I'd store `ast.imports.length` in a separate variable